bot.py

```diff
@@ -4,37 +4,31 @@
 import discord
 from discord.ext import commands
 from dotenv import load_dotenv
-GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
 
+load_dotenv()
+
+GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
 g = Github(GITHUB_TOKEN)
```

Comment:
:bike: (bike means bikeshedding or nitpick) I'd use `github` or `gh` instead of `g` (again, it's a nitpick haha)